ee/clickhouse/models/test/test_cohort.py

```diff
@@ -728,14 +722,8 @@ def test_cohortpeople_with_cyclic_cohort_filter(self):
         cohort1.groups = [{"properties": [{"key": "id", "type": "cohort", "value": cohort1.id}]}]
         cohort1.save()
 
-        cohort1.calculate_people_ch(pending_version=0)
-
-        count_result = sync_execute(
-            "SELECT count(person_id) FROM cohortpeople where cohort_id = %(cohort_id)s", {"cohort_id": cohort1.pk}
-        )[0][0]
-        self.assertEqual(count_result, 2)
+        self.assertRaises(ValueError, lambda: cohort1.calculate_people_ch(pending_version=0))
```

Comment:
@neilkakkar is cyclic not supposed to fail here?

Reply:
Indeed, this is correct, it's just that old query ignored it, so I reverted this back for the previous PR.